src/SiteProcess.php

```diff
@@ -50,14 +50,28 @@ public function __construct(AliasRecord $siteAlias, TransportInterface $transpor
     }
 
     /**
-     * @inheritdoc
+     * @return string
      */
-    public function setWorkingDirectory($cwd)
```

Comment:
I'm ambivalent about renaming this method. Your name is better. We're on the hook for a major version bump anyway. I have no reason for feeling reluctant.

Reply:
The diff is misleading I didn't rename it. I removed the our implementation of the method, so now the parent's is in effect I added setCd() and getCD()

Reply:
Oh I see.